### PR TITLE
Create gaps.subdomain.conf.sample

### PR DESCRIPTION
**Pull request by trolley**
For container: https://hub.docker.com/r/housewrecker/gaps

Also fix small typo in portainer.subdirectory.conf.sample.

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]
	

##  Thanks, team linuxserver.io

### CODE REVIEW

**Comment by trolley** [comment]
Subfolder sample was added in PR #325.

File: portainer.subfolder.conf.sample [hunk 1]
```diff
@@ -25,7 +25,7 @@ location ^~ /portainer/ {
     proxy_pass $upstream_proto://$upstream_app:$upstream_port;
 
     rewrite /portainer(.*) $1 break;
-    proxy_hide_header X-Frame-Options; # Possibly nott needed after Portainer 1.20.0
+    proxy_hide_header X-Frame-Options; # Possibly not needed after Portainer 1.20.0
```
Comment (drizuid) [inline portainer.subfolder.conf.sample:28]: "possibly" doesn't instill much confidence, please thoroughly test, we won't support a change that could break users.

Comment (drizuid) [inline portainer.subfolder.conf.sample:28]: this correction was made sep1, 2020

Comment (trolley) [reply portainer.subfolder.conf.sample:28]: I'm not sure what you're referring to. This file in the `master` branch currently has `nott` misspelled: https://github.com/linuxserver/reverse-proxy-confs/blob/master/portainer.subfolder.conf.sample

**Review by drizuid — CHANGES_REQUESTED** [review]
disregard, another typo was missed

File: portainer.subfolder.conf.sample (hunk 1 above)
Comment (drizuid) [reply portainer.subfolder.conf.sample:28]: i stand corrected, i missed that one

**Comment by trolley** [comment]
> please remove the portainer config as it's superfluous per [54ede40#diff-b0f0b1ac69c24158e8d491aa7f372ec5fbdf57ad0a9423e94c334dc44b27bf66](https://github.com/linuxserver/reverse-proxy-confs/commit/54ede409381b58e6b13d310fa9af623f42948072#diff-b0f0b1ac69c24158e8d491aa7f372ec5fbdf57ad0a9423e94c334dc44b27bf66)

That's a different line. In `master` the same comment appears on lines 28 and 40, and I'm just correcting the typo on line 28.

If you'd prefer a separate PR for that change I'm good to do that though.

**Review by drizuid — APPROVED** [review]

**Comment by trolley** [comment]
Thanks @drizuid.

**Pull request merged**